### PR TITLE
Microservice access configuration docking

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
Microservice access configuration docking

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     Microservice access configuration page        |
| 🇨🇳 中文    |      微服务接入配置页面        |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/modules/msp/pages/configuration/index.tsx [hunk 1]
```diff
@@ -74,87 +67,206 @@ const ItemRender = ({ title, children }: IProps) => {
...
+  const columns = [
+    { title: i18n.t('msp:accessKey'), dataIndex: 'accessKey' },
+    { title: i18n.t('create time'), dataIndex: 'createTime' },
+    { title: i18n.t('msp:last use time'), dataIndex: 'lastUsetime' },
+    {
+      title: i18n.t('application:operation'),
+      dataIndex: 'operation',
+      render: (_, record: { key: React.Key }) => (
+        <div className="ant-table-cell table-operations ant-table-cell-ellipsis">
+          <a onClick={getDetail} className="table-operations-btn">
```
Comment (Zero-Rock) [inline shell/app/modules/msp/pages/configuration/index.tsx:131]: getDetail doesn't need parameters ？

File: shell/app/modules/msp/pages/configuration/index.tsx [hunk 2]
```diff
@@ -74,87 +67,206 @@ const ItemRender = ({ title, children }: IProps) => {
...
+      ),
+    },
+  ];
+  // 查看AccessKey详情
+  const getDetail = () => {
+    setKeyDetailShow(true);
+    getDetailKey.fetch({
+      accessKeyId: 'M0Dy6Ia55juX4j8xmR0zHl2W',
+    });
+    const mock = getDetailKey.useData();
```
Comment (Zero-Rock) [inline shell/app/modules/msp/pages/configuration/index.tsx:147]: don't use Hooks in nested functions

File: shell/app/modules/msp/pages/configuration/index.tsx (hunk 1 above)
Comment (wuhuiluo) [reply shell/app/modules/msp/pages/configuration/index.tsx:131]: need,accessKeyId

File: shell/app/locales/en.json [hunk 3]
```diff
@@ -2191,6 +2192,7 @@
     "please enter http status code ": "please enter http status code ",
     "please enter your email address, separated by comma": "please enter your email address, separated by “,”",
     "please key in numbers": "please key in numbers",
+    "createAccessKey info": "please save or send the AccessKey information to the corresponding user in time. The information cannot be obtained again after the singing is closed, but you can create a new AccessKey at any time",
```
Comment (Zero-Rock) [inline shell/app/locales/en.json:2195]: after setting the alias manually, execute i18n

**Comment by daskyrk** [comment]
Add changeLog

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+      languages: newLanguages,
+      lang: newLanguages?.[0].type,
+      strategy: type,
+    });
+  };
+  const handleChangeLang = (type: string) => {
+    updater.lang(type);
+  };
+
+  const columns: Array<ColumnProps<any>> = [
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:133]: Array<ColumnProps<CONFIGURATION.IAllkeyData>>

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+    }
+  };
+
+  const deleteKey = async (id: string) => {
+    if (accressPerm.deleteAccessKey) {
+      await deleteDetailKey.fetch({
+        id,
+      });
+      message.success(i18n.t('application:deleted successfully'));
+      await getAllkey.fetch({
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:198]: the pageNo of there is 1, but the pageNo in the state is not updated

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+          <IconCopy size="14" />
+          <Copy selector=".container-key" copyText={`${createKeyInfo?.accessKey}\n${createKeyInfo?.secretKey}`}>
+            {i18n.t('copy')}
+          </Copy>
+        </div>
+      </Modal>
+
+      <Button className="m4 add-key" type="primary" onClick={createAccessKey}>
+        {i18n.t('msp:create AccessKey')}
+      </Button>
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:321]: when there is no permission, this button should be disabled

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 7]
```diff
@@ -0,0 +1,356 @@
...
+      dataIndex: 'operation',
+      key: 'operation',
+      render: (_: unknown, record: CONFIGURATION.IAllkeyData) => (
+        <div className="table-operations">
+          <a onClick={() => getDetail(record.id)} className="table-operations-btn">
+            {i18n.t('dcos:see details')}
+          </a>
+          <Popconfirm onConfirm={() => deleteKey(record.id)} title={`${i18n.t('common:confirm to delete')}?`}>
+            <a className="table-operations-btn">{i18n.t('application:delete')}</a>
+          </Popconfirm>
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:153]: when there is no permission, hide the delete button

File: shell/app/modules/msp/env-setting/configuration/type-select.scss [hunk 8]
```diff
@@ -7,3 +7,8 @@
     cursor: pointer;
   }
 }
+
+.container-key {
+  background-color: #dbeafe;
+  padding: 25px;
+}
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/type-select.scss:14]: can't tailwind be used here

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 9]
```diff
@@ -0,0 +1,356 @@
...
+
+      {!languages ? (
+        <EmptyHolder relative />
+      ) : (
+        <ItemRender title={i18n.t('msp:choose the language you want to connect')}>
+          <TypeSelect<LangItem> value={lang} list={languages || []} onChange={handleChangeLang} />
+        </ItemRender>
+      )}
+
+      <div className="container pt-4 pl-4 pb-4 pr-4 mt-2 rounded-sm">
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:349]: use p-4

File: shell/app/modules/msp/env-setting/configuration/index.scss [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+  font-weight: bold;
+}
+
+.container {
+  border: 1px solid $color-border;
+  height: 100%;
+  .info-data {
+    font-size: 14px;
+  }
+}
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.scss:11]: use tailwind

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 11]
```diff
@@ -0,0 +1,356 @@
...
+        ]}
+      >
+        <div className="rounded-sm container-key text-gray mb-2">
+          <div className="flex items-center mb-2">
+            <span>accessKey ID</span>
+            <span className="ml-32">{keyDetailInfo?.accessKey}</span>
+          </div>
+          <div className="flex items-center">
+            <span>accessKey Secret</span>
+            <span className="ml-24">{accressPerm.viewAccessKeySecret ? keyDetailInfo?.secretKey : '***'}</span>
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:265]: accressPerm.viewAccessKeySecret.pass

File: shell/app/common/utils/axios-config.ts [hunk 12]
```diff
@@ -68,8 +68,16 @@ export const initAxios = () => {
...
       const { data } = response.data;
-      if ('list' in data && 'total' in data && data.list === null) {
-        data.list = [];
+      // const { pageNo } = response.config.params;
+      // if (pageNo) {
+      //   if ('list' in data && 'total' in data && data.list === null) {
+      //     data.list = [];
+      //   }
+      // }
+      if (typeof data === 'object') {
```
Comment (Zero-Rock) [inline shell/app/common/utils/axios-config.ts:77]: If data is null, there will be error

File: shell/app/common/utils/axios-config.ts [hunk 13]
```diff
@@ -68,8 +68,16 @@ export const initAxios = () => {
       }
       // if paging list is null, transform to array
       const { data } = response.data;
-      if ('list' in data && 'total' in data && data.list === null) {
-        data.list = [];
+      // const { pageNo } = response.config.params;
+      // if (pageNo) {
+      //   if ('list' in data && 'total' in data && data.list === null) {
+      //     data.list = [];
+      //   }
```
Comment (Zero-Rock) [inline shell/app/common/utils/axios-config.ts:75]: remove useless code

**Pull request merged**